src/bananas/environment.py

```diff
@@ -264,6 +275,17 @@ def get_bool(self, key: str, default: None = None) -> Optional[bool]:
     def get_bool(self, key: str, default: object = None) -> object:
         return self.parse(parse_bool, key, default=default)
 
+    @overload
+    def get_path(self, key: str, default: Path) -> Path:
+        ...
+
+    @overload
+    def get_path(self, key: str, default: None = None) -> Optional[Path]:
+        ...
+
+    def get_path(self, key: str, default: object = None) -> object:
```

Comment:
```suggestion
    def get_path(self, key: str, default: Optional[Path] = None) -> Optional[Path]:
```

Would be more correct, but I see this is following the other implementations here.

Reply:
Ah I see, was a bit difficult cumbersome to figure out what's going on here but now I see it makes sense :slightly_smiling_face: